de.js

```diff
@@ -1006,8 +1006,8 @@ server.message.kicked {0} wurde gekickt
 server.message.banned {0} wurde gebannt
 server.message.votekicked {0} wurde per vote gekickt
 server.message.arrested {0} wurde vom K.P.D verhaftet
-event.easter.golden.spawn A Golden egg is near
-event.easter.golden.despawn A Golden egg has gone
-event.easter.found.new {0} found {1}/{2} Eggs
-event.easter.found.old You already discovered this egg!
-event.easter.completed {0} completed the Egg hunt!
+event.easter.golden.spawn Ein goldenes Ei ist nah
+event.easter.golden.despawn Ein goldenes Ei ist verschwunden
+event.easter.found.new {0} {1}/{2} Eier gefunden
+event.easter.found.old Du hast dieses Ei bereits gefunden!
+event.easter.completed {0} die Eiersuche beendet!
```

Comment:
{0} is a placeholder, this doesnt make sense in german. put "hat die Eiersuche abgeschlossen!"